core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -58,6 +58,30 @@ object BrokerIdsZNode {
   def encode: Array[Byte] = null
 }
 
+class BrokerInfo(id: Int,
+                 host: String,
+                 port: Int,
+                 advertisedEndpoints: Seq[EndPoint],
+                 jmxPort: Int,
+                 rack: Option[String],
+                 apiVersion: ApiVersion) {
+
+  def ID(): Int = {
```

Comment:
Instead of defining this method. We can just make id val in the constructor.

Reply:
done